multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -110,4 +112,11 @@ storage:
       group:
         name: root
       contents:
-        local: create-cex-device.sh
\ No newline at end of file
+        local: create-cex-device.sh
+
+    - path: /dev/vfio/0
+      mode: 0666
+      user:
+        name: root
+      group:
+        name: root
```

Comment:
This will not do the right thing. We can't change a device node's permissions from Ignition directly.

Let's move it to `create-cex-device.sh` instead?

Reply:
Will do that.